sales.html

```diff
@@ -2,15 +2,11 @@
 <html>
   <head>
     <Title>Sales Projections for Pat""s Stores</Title>
+    <a href="index.html"></a>
```

Comment:
This anchor tag will need to be moved into the body element in order to render to the page.